aten/src/TH/cmake/FindMKL.cmake

```diff
@@ -37,16 +37,9 @@ SET(INTEL_MKL_SEQUENTIAL OFF CACHE BOOL
   "Force using the sequential (non threaded) libraries")
 
 # Checks
-CHECK_TYPE_SIZE("void*" SIZE_OF_VOIDP)
-IF ("${SIZE_OF_VOIDP}" EQUAL 8)
-  SET(mklvers "em64t")
-  SET(iccvers "intel64")
-  SET(mkl64s "_lp64")
-ELSE ("${SIZE_OF_VOIDP}" EQUAL 8)
-  SET(mklvers "32")
-  SET(iccvers "ia32")
-  SET(mkl64s)
-ENDIF ("${SIZE_OF_VOIDP}" EQUAL 8)
+SET(mklvers "em64t")
+SET(iccvers "intel64")
+SET(mkl64s "_lp64")
```

Comment:
...no 32-bit?

Reply:
Yeah, @soumith said that we only consider 64bit systems now.